account-kit/react/src/components/auth/context.ts

```diff
@@ -14,6 +15,7 @@ export type AuthStep =
 type AuthContextType = {
   authStep: AuthStep;
   setAuthStep: (step: AuthStep) => void;
+  uiConfig?: AlchemyAccountsUIConfig;
```

Comment:
@moldy530 I wasn't entirely sure if this was the right place for this, but it seems to make the most sense because it's internal state related to the auth flow/UI, and introducing a new context just for the UI config felt way too overkill

Reply:
yea I wonder if instead we should add an export for `createConfig` in `@account-kit/react` which wraps the `createConfig` call from `core` but adds a param for `uiConfig` now. That way it's part of the return type there and you shouldn't need `useAuthContext` to get the `uiConfig` and instead you can just use the `useAccountContext` or w/e it was called (or maybe add a hook called `useUiConfig`)

Reply:
that way you can just use that hook everywhere you need the ui config and you don't have to prop drill it anywhere